### PR TITLE
ROADMAP.md: Add isolated item for WebRTC

### PR DESCRIPTION
**Pull request by mxinden**
- Adds an _evolve_ section roadmap item for WebRTC.
- Removes mention of WebRTC from other items to not double track.

_As discussed in Paris. Will follow with additional pull requests._

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -290,6 +287,25 @@ the [libp2p test-plans repository].
...
+**Status**: In progress
+
+**What?** WebRTC is a transport protocol supported by all major browsers. Those
+browsers allow the establishment of connections to remote endpoints that don't
+have a TLS certificate signed by a trusted certificate authority. In addition
+WebRTC includes hole punching capabilities.
+
+**Why?** In most p2p networks the majority of nodes does not have a signed TLS
+certificate. With WebRTC browsers will thus be able to connect to these
+previously unreachable nodes. In addition, being able to hole punch allows
```
Comment (marten-seemann) [inline ROADMAP.md:301]: This should probably reference WebTransport.

**Comment by marten-seemann** [comment]
Should we merge the WebTransport PR first?

**Comment by mxinden** [comment]
> Should we merge the WebTransport PR first?

I am fine either way. Sounds good to me @marten-seemann.

File: ROADMAP.md [hunk 2]
```diff
@@ -290,6 +287,25 @@ the [libp2p test-plans repository].
...
+### WebRTC
+
+**Status**: In progress
+
+**What?** WebRTC is a transport protocol supported by all major browsers. Those
+browsers allow the establishment of connections to remote endpoints that don't
+have a TLS certificate signed by a trusted certificate authority. In addition
+WebRTC includes hole punching capabilities.
+
+**Why?** In most p2p networks the majority of nodes does not have a signed TLS
```
[the remark this answers is not in the document]
Comment (mxinden) [reply ROADMAP.md]: See https://github.com/libp2p/specs/pull/414/commits/df4fb0cc85f6f30079b4c5c0c72a8b354ead2a20.

File: ROADMAP.md (hunk 1 above)
Comment (mxinden) [reply ROADMAP.md:301]: Good idea. Done with https://github.com/libp2p/specs/pull/414/commits/df4fb0cc85f6f30079b4c5c0c72a8b354ead2a20.

**Comment by mxinden** [comment]
> Should we merge the WebTransport PR first?

With https://github.com/libp2p/specs/pull/407 merged, does this look good to you @marten-seemann?

**Pull request merged**